### DISCUSSION TITLE
Practical series 4

### DISCUSSION THREAD

**Discussion by Will-214** [IBAHCM/RPiR#295] [Archive] [answered]
Hi @soniamitchell @richardreeve  I'm trying to edit the documentation of my package and I include RPIR as a dependency. But I get a message in the console telling me that RPIR does not exist. If I then run 'install.packages("RPIR")' it says:
"Warning in install.packages :
  package ‘RPIR’ is not available for this version of R"

Once RPIR is removed from my description and package documentation file 'devtools::install()' seems to run fine. 
I think it does the same for dplyr as well. Is this a big issue? Or am I doing something I'm not supposed to be doing?

Thanks

**Comment by soniamitchell** [comment, marked as answer]
Hi @Will-214, did you follow these instructions? https://ibahcm.github.io/RPiR/articles/pages/packages_guide.html#add-dependencies

Remember that RPiR is not available on CRAN, only on GitHub, so you should use `usethis::use_dev_package("RPiR", "Imports", remote = "IBAHCM/RPiR")` to add it as a dependency. Also, you should use `devtools::install_github()` rather than `install.packages()` to install it.

**Reply by richardreeve** [reply]
Yes, as Sonia says we have been using `devtools::install_github("IBAHCM/RPiR")` to install RPiR because it's not on CRAN.

In general, you may find that you need to install GitHub packages (using `devtools::install_github()`) in advance before you add them as dependencies to other packages using `usethis::use_dev_package()`.

**Reply by Will-214** [reply]
OK, thanks very much, I'll give this a try! Seems to have worked for RPIR
